src/Anonymization/Anonymizer/Core/DateAnonymizer.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core;
+
+use Doctrine\DBAL\Platforms\SqlitePlatform;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\AbstractAnonymizer;
+use MakinaCorpus\DbToolsBundle\Attribute\AsAnonymizer;
+use MakinaCorpus\QueryBuilder\Query\Update;
+
+#[AsAnonymizer(
+    name: 'date',
+    pack: 'core',
+    description: <<<TXT
+    Anonymize a column by changing the date it contains.
+    You can either choose a 'min' and a 'max' date, case in which a random date will
+    be selected between these bounds, or alternatively set a 'delta' which must be
+    a valid date interval string (e.g. "1 week", "1 day 10 hours", ...).
+    You should set the 'format' (default: 'datetime') value as this anonymizator
+    can work with 'datetime' or 'date' formats.
+    TXT
+)]
+class DateAnonymizer extends AbstractAnonymizer
+{
+    #[\Override]
+    public function anonymize(Update $update): void
```

Comment:
Can you rebase `main` then implement the `::validateOptions` method ?

Reply:
Done !